mithril.js

```diff
@@ -1706,16 +1706,16 @@
 		var currentTarget = e.currentTarget || e.srcElement
 		var args
 
+		while (currentTarget && !currentTarget.href) {
+			currentTarget = currentTarget.parentNode
+		}
```

Comment:
@adriengibrat 

I'm not requiring you change it (this is just another way of doing the same thing), but [checking `href` isn't exactly a silver bullet](https://github.com/lhorie/mithril.js/pull/1009#discussion_r58303353). I just thought I'd point that out for you.